sentry-samples/sentry-samples-logback/src/main/resources/logback.xml

```diff
@@ -11,6 +11,7 @@
       <debug>true</debug>
       <!-- NOTE: Replace the test DSN below with YOUR OWN DSN to see the events from this app in your Sentry project/dashboard -->
       <dsn>https://502f25099c204a2fbf4cb16edc5975d1@o447951.ingest.sentry.io/5428563</dsn>
+      <mdcTag>userId</mdcTag>
```

Comment:
should this sample also show how to set multiple?

Reply:
is `<mdcTag>` correct here? or should it be `<mdcTags>`?

Reply:
It is correct. under the hood it calls `addMdcTag`

Reply:
Sample updated.